inc/classes/class_auth.php

```diff
@@ -635,7 +635,7 @@ public function update_visits($frmwrkmode = "")
      * @param int $userid
      * @return void
      */
-    public function set_cookie_pw($userid)
+    private function set_cookie_pw($userid)
```

Comment:
Is called in usrmgr/changepw.php line 22. Would also cause a problem.

Reply:
My IDE didnt catched this because of all the global variables. Fixed.